packages/appeal-reply-service-api/src/__tests__/notify.spec.js

```diff
@@ -0,0 +1,83 @@
+const axios = require('axios');
+const NotifyBuilder = require('@pins/common/src/lib/notify/notify-builder');
+const notify = require('../lib/notify.js');
+const config = require('../lib/config');
+
+jest.mock('axios');
+jest.mock('@pins/common/src/lib/notify/notify-builder', () => {
+  const mock = {
+    setNotifyClient: jest.fn(() => mock),
+    setTemplateId: jest.fn(() => mock),
+    setTemplateVariable: jest.fn(() => mock),
+    setTemplateVariablesFromObject: jest.fn(() => mock),
+    addFileToTemplateVariables: jest.fn(() => mock),
+    setDestinationEmailAddress: jest.fn(() => mock),
+    setEmailReplyToId: jest.fn(() => mock),
+    setReference: jest.fn(() => mock),
+    sendEmail: jest.fn(() => Promise.resolve('done')),
+  };
+  return mock;
```

Comment:
why can't this be short hand?

because it is self referencing - jest fn's return self

Reply:
cc @davehlewis re: https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/1136#discussion_r637919650